documentation/mgmt_ide-setup.asciidoc

```diff
@@ -1,69 +1,69 @@
 +
 image::images/howtos/ide-setup/File_Structure.png[Install directory after executing setup.bat]
 +
-2. As devonfw ide by default does not install eclipse anymore, we need to setup eclipse first by executing `devon eclipse`
+2. As devonfw IDE by default does not install eclipse anymore, we need to setup eclipse first by executing `devon eclipse`
 3. The next step is to open a console and execute the following command: `+devon project setup cobigen+`
-This downloads all the necessary files for Cobigen development. Build the project by running the build.sh located in the workspaces/main/cobigen folder within git bash.
-4. Optional: If you are making use of SSH private key authentication working with Git on GitHub, you can change the HTTPS default setup git remote url by executing `git remote set-url origin git@github.com:devonfw/cobigen.git` (possibly with `git@github.com:<your user>/cobigen.git` in case you want to work on your fork)
+This downloads all the necessary files for CobiGen development. Build the project by running the build.sh located in the `workspaces/main/cobigen` folder within git bash.
+4. Optional: If you are making use of SSH private key authentication working with Git on GitHub, you can change the HTTPS default setup git remote URL by executing `git remote set-url origin git@github.com:devonfw/cobigen.git` (possibly with `git@github.com:<your user>/cobigen.git` in case you want to work on your fork)
 5. Now open eclipse using the eclipse-main.bat file or by executing `devon eclipse` on the console and import the CobiGen projects you want to work on
 6. Switch to the "Project Explorer" view (Window->Show View->Project Explorer). This extra step is required because an import from the default view "Package Explorer" doesn't work properly.
-7. Click on File->import...->Maven->Existing Maven Project and entering {Install directory}/workspaces/main/cobigen (Should be the default location when clicking on "Browse...")
+7. Click on File->import...->Maven->Existing Maven Project and entering {Install directory}`/workspaces/main/cobigen` (Should be the default location when clicking on "Browse...")
 8. After you have have finished your installation run a maven update. To do so right click on a project, select maven and afterwards update project. Select all projects and the checkbox Force Update of Snapshot/Releases.
 You might be asked to install some Tycho-plugins. You need those, if you want to debug eclipse-plugins. 	
 
 == Eclipse Plugin Installation
 
-For some parts of Cobigen, you will have to have additional plugins installed.
+For some parts of CobiGen, you will have to have additional plugins installed.
 
 === Plugin development
 
-If you want to develop Cobigen plugins (openapiplugin, javaplugin etc.) you need to have the eclipse PDE plugin available. 
+If you want to develop CobiGen plugins (OpenAPI plugin, Java plugin etc.) you need to have the eclipse PDE plugin available. 
```

Comment:
please just allow "Plug-in" as term - plugin should not be valid to make it consistent

Reply:
I think, plugin is valid word and not detected as spell mistake. Also plugin is used at many occurrences... changing it at every place will take too much effort as we cannot use replace all blindly. What do you think can we keep plugin in document or should it be replaced by Plug-in?